### ISSUE TITLE
Simple importance system for people

### ISSUE THREAD

**Issue by mikron-ia** [mikron-ia/rpg-hub#134]
A system that considers:

- how often someone visits a certain character (via #97 or #78)
- how personal it is to them (give high value to own character, lower to the party)
- newcomer bonus (new character receives a bonus that may degrade over time)
- manual GM input (highlighting important people)

and provides a **personalised** information of how important is the given character to the user. This, in turn, is a base for descending sort of the characters in the character screen.

**Comment by mikron-ia** [comment]
This, of course, is about `Character` models. "People" are of old nomenclature, changed in #124.

**Comment by mikron-ia** [comment]
This should also work for `Group`, as per #379 - which should be more or less automatic, considering that `ImportancePack` is linked to both. This is, thus, not much of an issue.

The real problem with this task is that it was created long ago, when the whole structure of the project was different. Addressing particulars:

* how often someone visits a certain character - this is actually viable, but may require a few new mechanisms; existing logs do not have sufficient information; this is still possible, but modifying `Seen` mechanism would likely be needed
* how personal it is to them (give high value to own character, lower to the party) - this will require a dedicated party mechanism; nothing like this is planned except maybe #193, potentially putting this task on indefinite hold; at the same time, this can be replaced with a manual flag, use of a importance category, or just giving a boost to everything that has an `active` character sheet (via #332 and #339)
* newcomer bonus (new character receives a bonus that may degrade over time) - this is easy
* manual GM input (highlighting important people) - this is already done via importance category

In other words, this needs some re-interpretation and is likely blocked. Considering moving to later milestone depending on how the above list is resolved.

**Comment by mikron-ia** [comment]
Revised completion:

* visitation frequency - not done, changes to `Seen` not implemented
* personal value - can use association bonus via the `Group` - not implemented yet
* newcomer bonus - done through one of the `Importance` parameters
* manual GM input - done through `ImportanceCategory`
